test/functional/rpc_net.py

```
@@ -394,6 +395,28 @@ def test_sendmsgtopeer(self):
         node.sendmsgtopeer(peer_id=0, msg_type="addr", msg=zero_byte_string.hex())
         self.wait_until(lambda: len(self.nodes[0].getpeerinfo()) == 0, timeout=10)
 
+    def test_getaddrmaninfo(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_getaddrmaninfo`

Add return type annotation: `None`

(ANN201)